src/commentsPopup.test.js

```diff
@@ -0,0 +1,7 @@
+import { addCommentCount } from './commentsPopup.js';
+
```

Comment:
- [ ] Kindly ensure that the test case covers all the pure functions used in this project

Reply:
Done